VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java

```diff
@@ -161,6 +161,7 @@ public void onClick(View v) {
                 threadBody.setRawBody(body);
                 threadBody.setTopicId(((DiscussionTopicDepth) topicsSpinner.getSelectedItem()).getDiscussionTopic().getIdentifier());
                 threadBody.setType(discussionQuestion);
+                threadBody.setFollowing(true);  // MA-1380
```

Comment:
You can also link to the JIRA issue in the comment if you want, but the explanation should be there.
> On 30-Sep-2015, at 3:56 pm, Ibraheem Zaman <ibraheem@edx.org> wrote:
> 
> In that case, just add a comment explaining that we’re emulating the behaviour of the website by auto-following new posts. Also, since we’re using "following=true” as the default, maybe we could just set it as the default value for the variable (and move the comment there)?
>> On 30-Sep-2015, at 3:45 pm, Mi@N Kh@LiD <notifications@github.com <mailto:notifications@github.com>> wrote:
>> 
>> In VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java <https://github.com/edx/edx-app-android/pull/378#discussion_r40780399>:
>> 
>> > @@ -161,6 +161,7 @@ public void onClick(View v) {
>> >                  threadBody.setRawBody(body);
>> >                  threadBody.setTopicId(((DiscussionTopicDepth) topicsSpinner.getSelectedItem()).getDiscussionTopic().getIdentifier());
>> >                  threadBody.setType(discussionQuestion);
>> > +                threadBody.setFollowing(true);  // MA-1380
>> since we are sending a hardcoded value in this case, so, I thought it would be better to explain the reason for it 
>> 
>> —
>> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/378/files#r40780399>.
>> 
>